paasta_tools/kubernetes/application/controller_wrappers.py

```diff
@@ -115,8 +130,8 @@ def ensure_pod_disruption_budget(
         self, kube_client: KubeClient
     ) -> V1beta1PodDisruptionBudget:
         pdr = pod_disruption_budget_for_service_instance(
-            service=self.item.service,
-            instance=self.item.instance,
+            service=self.kube_deployment.service,
+            instance=self.kube_deployment.instance,
             min_instances=self.soa_config.get_desired_instances()
```

Comment:
Hm, I'm wondering how we want to handling autoscaling for pod disruption budgets. As is, we would be willing to go pretty low because get_desired_instances returns min_instances if autoscaling is enabled. But we also can't update the PDB all the time.

Can we use a percentage instead of min_instances? https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget